Source/gendung.h

```diff
@@ -20,14 +22,54 @@ enum dungeon_type : int8_t {
 	DTYPE_NONE = -1,
 };
 
+enum lvl_entry : uint8_t {
+	ENTRY_MAIN,
+	ENTRY_PREV,
+	ENTRY_SETLVL,
+	ENTRY_RTNLVL,
+	ENTRY_LOAD,
+	ENTRY_WARPLVL,
+	ENTRY_TWARPDN,
+	ENTRY_TWARPUP,
+};
+
+enum {
+	// clang-format off
+	DLRG_HDOOR     = 0x01,
+	DLRG_VDOOR     = 0x02,
+	DLRG_CHAMBER   = 0x40,
+	DLRG_PROTECTED = 0x80,
+	// clang-format on
+};
+
+enum {
+	// clang-format off
+	BFLAG_MISSILE     = 0x01,
+	BFLAG_VISIBLE     = 0x02,
+	BFLAG_DEAD_PLAYER = 0x04,
+	BFLAG_POPULATED   = 0x08,
+	BFLAG_MONSTLR     = 0x10,
+	BFLAG_PLAYERLR    = 0x20,
+	BFLAG_LIT         = 0x40,
+	BFLAG_EXPLORED    = 0x80,

```

Comment:
Same as the other one. Consider bit-shifting syntax to make it clear each position on the bitmask.